package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "24.5.3",
+  "version": "24.5.4",
```

Comment:
This was _just_ updated already to 4: https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/package.json#L3

Reply:
Updated to 24.5.5